content/articles/understanding-different-types-of-menus-in-android-with-kotlin/index.md

```diff
@@ -0,0 +1,261 @@
+### [Android] Understanding Different Types of Menus in Android with Kotlin.
+#### What is a menu as used in android
+Menus are fundamental user interface elements that are frequently used to store actions that would otherwise be buried.
+These hidden commands could be accessible by:
+- Click a button.
+- Using gestures such as long-pressing an image.
+#### Reasons of having menus as a user interface component
+- To free up screen space.
+- To perform operations that are mostly required by users who are not directly represented on the screen.
+- To provide a capability that allows shifting to various parts of the application that are not easily available in the presence of user interface display.
+#### Prerequisites
+- Have android studio installed
+- Have basic knowledge of kotlin programming language.
+- Have a basic knowledge of android.
+
+#### Types of Menus Used in Android
+Based on the different contexts and content of the application, Different menu types are explained which are used to provide the best user experience. Here are the types of menus.
+
+1. Options menu
+
+Options menus are menus that present actions and other options that are directly relevant to the application's current activity or context.
+The following are some examples of choices menus:
+- Settings
+- Share
+- Search
+- Copy link
+- Help
+- History
+- Privacy
+
+#### Creating options menu
+On the resource, the folder creates a menu directory and within the directory create an options menu file the includes the following implementation to build your menu items.
```

Comment:
Paraphrase this to indicate that you're the one to create a menu directory, not the folder as mentioned above.